analysers/analyser_merge_parapharmacy_IT.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python
+#-*- coding: utf-8 -*-
+
+###########################################################################
+##                                                                       ##
+## Copyright 2020                                                        ##
```

Comment:
Add you name or pseudo, you cannot make anonymous copyright. Or make this public domaine of you want.